src/widgets/events/constants.ts

```diff
@@ -0,0 +1,13 @@
+import dayjs from 'dayjs';
+import { getActualData } from '@/shared/helpers/getActualData.ts';
+import { events } from 'data';
+
+export const displayedCardsQuantity = 2;
+export const actualEvents = getActualData({
+  data: events,
+  staleAfter: 3,
+});
+
+export const nearestEvents = actualEvents.slice(0, displayedCardsQuantity);
+
+export const rsLifetime = dayjs().diff('2013', 'year');
```

Comment:
I agree

Reply:
@ansivgit but it's not hardcoded or am i missing something?

Reply:
@coderabbitai explain what do you mean by "Hardcoded 2013 may need future updates"

Reply:
@coderabbitai this year will never be changed because it's stands for year of organization founding. But probably a good point to keep it as a global constant for the entire codebase

Reply:
@SpaNb4 It's really better to put this into constants.

Reply:
do you propose to make the string with 2013 as a constant?

Reply:
I suppose it should be global constant